ivy_tests/test_ivy/helpers.py

```diff
@@ -1114,6 +1195,28 @@ def test_function(
         ivy.unset_backend()
         raise e
     ivy.unset_backend()
+    # gradient test
+    if (
+        test_gradients
+        and not fw == "numpy"
+        and all(as_variable_flags)
+        and not any(container_flags)
+        and not any(native_array_flags)
```

Comment:
native arrays should be fine actually I think, or am I missing something? Native arrays can also be variables.

Reply:
Actually, native arrays further exceed the deadline to reach nearly 1700ms, so now I've removed the condition and have further increased the deadline to 2000ms.